### DISCUSSION TITLE
Can't access account

### DISCUSSION THREAD

**Discussion by mertdeveci5** [npm/feedback#1073] [General]
Hi - my registered email was cancelled with Google, I cannot access my account on npmjs and I cannot register. I need to access my invoices and account. Your support form in the website is dysfunctional. Can someone from the support team reach out to resolve this please?

**Comment by MylesBorins** [comment]
Please try and use the support form on the website one more time. If you run into issues again please post in here the error code you are getting and we'll figure out a solution

**Comment by JulieGunawan** [comment]
I have the same issues, and I tried to use the support form, but somehow, the support form also rejected my form and got an error "We are unable to verify the captcha", is there any way to contact you guys to get help on this? thank you

**Comment by mertdeveci5** [comment]
It does not work on the website whatever I do - that is why we reached out over here. Can you please help?
